src/mcp_server_my_mac/readers/load_conda_info.py

```
@@ -221,3 +222,57 @@ def load_conda_env_package_list(env_name: str):
             return f"Error listing packages in {env_name}: {result.stderr}"
     except Exception as e:
         return f"Error: {str(e)}"
+
+
+def load_gpu_available_mac_torch(env_name: str) -> bool:
+    conda_executable = find_conda_executable()
+    if not conda_executable:
+        logging.error("Conda executable not found")
```

Ruff (0.8.2)

Undefined name `logging`

(F821)